src/AutSoft.AspNetCore.Blazor/wwwroot/lib/nav.js

```diff
@@ -0,0 +1,150 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   hook.js                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mafaussu <mafaussu@student.42lyon.fr>      +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/06/30 13:00:00 by mafaussu          #+#    #+#             */
+/*   Updated: 2022/07/05 14:34:21 by mafaussu         ###   ########lyon.fr   */
+/*                                                                            */
+/* ************************************************************************** */
+
+const nav_state_time_key = '__tsd_nav_creation';
+let nav_prev_position;
+let nav_direction;
+let nav_position;
+let nav_history;
+
+function get_nav_position() {
+  return nav_position;
+}
+
+function get_nav_state_time_key() {
+  if (history.state)
+    return history.state[nav_state_time_key];
+
+  return nav_history[0].state[nav_state_time_key];
+}
+
+function init_nav() {
+  nav_direction = "initial page";
+  nav_position = 1;
+  nav_history = [{
+    url: document.location.href,
+    state: {
+      [nav_state_time_key]: new Date().getTime()
+    }
+  }
+  ];
+}
+
+init_nav();
+
+window.addEventListener('nav::tick', function (e) {
+
+})
```

Comment:
Ezt az eventlistenert lehet törölni sztem.

Reply:
Akkor a `nav_dispatch` függvény is törölhető?

Reply:
igen